dealership/vehicles.py

```diff
@@ -1,15 +1,53 @@
 class Vehicle(object):
     def __init__(self, maker, model, year, base_price, miles):
-        pass
+        self.maker = maker 
+        self.model = model
+        self.year = year 
+        self.base_price = base_price
+        self.miles = miles 
 
+# Sale multipliers
+# Car: 1.2
+# Motorcycle: 1.1
+# Truck: 1.6
 
+# Purchase multipliers
+# Car: 0.004
+# Motorcycle: 0.009
+# Truck: 0.02
+        
+# add dict to hold sale price and purchase price 
 class Car(Vehicle):
-    pass
-
+    sale_multiplier = 1.2
+    purchase_multiplier = 0.004
```

Comment:
Usually, when you have constant values you want to refer to them with named constants, for example in this case SALE_MULTIPLIER and PURCHASE_MULTIPLIER.

Reply:
I will Capitalize the Constants